docs/documentation/schedule/components/fares.md

```diff
@@ -53,19 +53,19 @@ This feature can be used to create rules that are useful to represent fares diff
 
 <div class="grid" markdown>
 
-This feature allows to define rules that enable fares differentiated when traveling from a specific group of stops to another. This can be useful to represent zone-based systems where a specific fare applies when traveling from one particular zone to another.
+Zone-Based Fares is used to represent zone-based systems where a specific fare applies when traveling from one particular zone to another. A zone is defined by a group of stops.
 
 | Files associated      | [fare_leg_rules.txt](/schedule/reference/#fare_leg_rulestxt)                            | [areas.txt](/schedule/reference/#areastxt)            | [stop_areas.txt](/schedule/reference/#stop_areastxt)     |
 |-----------------------|-----------------------------------------------|----------------------|--------------------|
 | **Fields associated** | fare_product_id<br>from_area_id<br>to_area_id | area_id<br>area_name | area_id<br>stop_id |
 
 </div>
 
-## Transfer Fares
+## Fares Transfers
```

Comment:
I think `Fares Transfers` should also be updated on the component Overview page (docs/schedule/components/index.md).

Reply:
I am confused by why there are 2 almost identical pages for the Fares component:
1. docs/documentation/schedule/components/fares.md
2. docs/schedule/components/fares.md

Page number 2 says `Transfer Fares` instead of `Fares Transfers`.

Reply:
That's because we haven't removed files from the previous document structure, so basically Components content is duplicated. "1. docs/documentation/schedule/components/fares.md" should be the most up-to-date.

Reply:
Got it, thanks! Let's make sure the links under the "Learn more about this feature" buttons on the overview page point to `docs/documentation/schedule/components/...`

Reply:
I didn't touch the overview page in this PR, I am going to open another PR to make modifications to it based on this PR